packages/manager/src/dev-tools/FeatureFlagTool.tsx

```diff
@@ -75,6 +84,9 @@ export const FeatureFlagTool = withFeatureFlagProvider(() => {
               </div>
             );
           })}
+          <button onClick={resetFlags} style={{ marginTop: 8 }}>
+            Reset default flags
```

Comment:
Can we reword this to `Reset to LaunchDarkly default flags` for more clarity?
```suggestion
            Reset to LaunchDarkly default flags
```

Reply:
I almost made this same comment, then deleted it, and asked for a code comment, ha. I do still think it would be nice. 

I was thinking of (1) more text requiring further height increase of the dev tools panel and (2) whether we'd want to specify LD if to save some trouble of updating it if we were to switch to another tool in the future - but really that's such an easy change.

Reply:
Same brain cell 🧠 . (1) We can shorthand to LD to save some width space (2) Agreed